app/models/metric/capture.rb

```diff
@@ -38,6 +39,12 @@ def self.alert_capture_threshold(target)
                 ::Settings.performance.capture_threshold_with_alerts.default)
   end
 
+  attr_accessor :ems, :interval_name
+  def initialize(ems = nil, interval_name = "realtime")
```

Comment:
Is a nil ems a valid default here?  Looks like it is required further down (at least for ems.zone and maybe `Metric::Targets.all_capture_targets(ems)`...not sure what that does with a nil arg yet)

Reply:
this object is not valid without an `ems`
But I feel we should always be able to create objects without args for things like specs

Reply:
fixed